blockchain/difficulty.go

```diff
@@ -286,7 +287,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 
 	// Limit the amount of adjustment that can occur to the previous
 	// difficulty.
-	actualTimespan := lastNode.timestamp.UnixNano() - firstNode.timestamp.UnixNano()
```

Comment:
Assuming the int64 holds milliseconds instead of nanoseconds, doesn't this change the value of actualTimespan with a factor of 1000? It is later compared with `b.minRetargetTimespan`, which is not changed in this PR.